core/commonMain/src/kotlinx/serialization/descriptors/SerialDescriptor.kt

```diff
@@ -5,11 +5,12 @@
 package kotlinx.serialization.descriptors
 
 import kotlinx.serialization.*
+import kotlinx.serialization.builtins.*
 import kotlinx.serialization.encoding.*
 
 /**
  * Serial descriptor is an inherent property of [KSerializer] that describes the structure of the serializable type.
- * The structure of the serializable type is not only the property of the type, but also of the serializer as well,
+ * The structure of the serializable type is not only the property of the type itself, but also of the serializer as well,
```

Comment:
```suggestion
 * The structure of the serializable type is not only the characteristic of the type itself, but also of the serializer as well,
```

I suggest replacing 'property' to avoid confusion with class properties